training/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
-spring.datasource.url=jdbc:postgresql://localhost:5432/dbname
-spring.datasource.username= POSTGRESUSERNAME
-spring.datasource.password= POSTGRESPASSWORD
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username= postgres
+spring.datasource.password= hola1234
```

Comment:
In the real-life, you should never leave credentials exposed in the code. You can use the _environment variables_ of IntelliJ to set these values.
You should go to `Run -> Edit Configurations -> Environmnet Variables` and paste them exactly as you have them in the application.properties separated by `;`
For example: `spring.datasource.username=sampleUsername;spring.datasource.password=samplePass`